tests/test_interpreter.py

```diff
@@ -54,7 +71,7 @@ def test_hello_world():
         {"role": "assistant", "message": hello_world_response},
     ]
 
-"""
+@pytest.mark.skip(reason="Math is hard")
```

Comment:
@KillianLucas I updated this to use `pytest`'s built-in `skip` functionality instead of just commenting it out.